client/signup/steps/design-type-with-atomic-store/index.jsx

```diff
@@ -12,15 +12,10 @@ import { includes, invoke } from 'lodash';
  */
 import config from 'config';
 import StepWrapper from 'signup/step-wrapper';
-import Card from 'components/card';
+import Tile from 'components/tile-grid/tile';
+import TileGrid from 'components/tile-grid';
 import { localize } from 'i18n-calypso';
 import { recordTracksEvent } from 'state/analytics/actions';
-import {
-	BlogImage,
-	PageImage,
-	GridImage,
-	StoreImage,
-} from '../design-type-with-atomic-store/type-images';
```

Comment:
Can we remove the entire file these are imported from?

https://github.com/Automattic/wp-calypso/blob/e9a8179b5a30238927d65c499fe5357018e99645/client/signup/steps/design-type-with-atomic-store/type-images.jsx#L1-L23

Reply:
👍  8ce9473bd06dee6874984db661f55d1f60bab94f